umbraco-cloud/upgrades/migrate-from-umbraco-10-to-11.md

```diff
@@ -0,0 +1,161 @@
+---
+description:
+  In this article we show how you can upgrade your Umbraco 10 Cloud project locally to Umbraco 11 and then migrate the project to a new Umbraco 11 project.
+---
+
+# Migrating a Umbraco 10 project to a Umbraco 11 project
+
+Due to a temporary bug in the hosting services, we do not recommend to directly upgrade to Umbraco 11.
+
+In this guide, we show how you can migrate your Umbraco 10 project to a new Umbraco 11 project on Umbraco Cloud instead.
+
+{% hint style="info" %}
+**Are you using any custom packages?**
+
+You will need to ensure the packages you use are available in Umbraco 11.
+
+**Breaking Changes**
+
+Make sure you know the [Breaking changes](../../umbraco-cms/fundamentals/setup/upgrading/version-specific/) in the latest version of Umbraco CMS.
+{% endhint %}
+
+## Prerequisites
+
+* Follow the **requirements** for [local development](../../umbraco-cms/fundamentals/setup/requirements.md#local-development).
+* A Umbraco Cloud project running **the latest Umbraco 10**
+* The .NET 7 Software Development Kit (SDK) installed locally.
+* A clean Umbraco 11 project.
+
+## Step 1: Enable .NET 7
+
+* Go to the Umbraco 10 project in the Umbraco Cloud portal.
+* Navigate to **Settings** -> **Advanced**.
+* Scroll down to the **Runtime Settings** section.
+* **Enable the latest version of .NET** for each environment on your Cloud project.
+
+<figure><img src="../../.gitbook/assets/runtime-settings.png" alt=""><figcaption><p>Runtime settings</p></figcaption></figure>
+
+## Step 2: Clone down your environment
+
+* Clone down the **Live** environment.
+* Build and run the [project locally](../set-up/working-locally.md#running-the-site-locally).
+* Log in to the backoffice.
+* Restore content from your Cloud environment.
+
+## Step 3: Upgrade the project locally using Visual Studio
+
+* Open your project in Visual Studio - use the `csproj` file in the `/src/UmbracoProject` folder.
+* Right-click your project solution in **Solution Explorer**.
+* Select **Properties**.
+
+<figure><img src="images/Solution-Explorer.png" alt=""><figcaption></figcaption></figure>
+
+* Select the **.Net 7** **Target Framework** drop-down in the **General** section of the **Application** tab as on your Cloud project.
```

Comment:
This is quite confusing 🙈 
Perhaps try to split this up to simplify the steps a bit?

Reply:
I've tried to simplify it, let me know what you think!